packages/default/scss/scrollview/_layout.scss

```diff
@@ -47,29 +47,6 @@
         }
     }
 
-    kendo-scrollview.k-scrollview-wrap.k-scrollview-multidot .k-scrollview-pageable > .k-button,
-    kendo-scrollview.k-scrollview.k-scrollview-multidot .k-scrollview-pageable > .k-button,
-    .k-widget.k-scrollview.k-scrollview-multidot .k-scrollview-nav > .k-link {
```

Comment:
Check if removing these styles will break the jQuery widget.

Reply:
I couldn't find the same class pattern on the jQuery website when inspecting their scrollView demos.